tutorials/interoperability/xcm-channels/index.md

```diff
@@ -1,20 +1,20 @@
 ---
 title: Tutorials for Managing XCM Channels
-description: Learn step-by-step how to establish unidirectional and bidirectional HRMP channels between parachains and system parachains using XCM.
+description: Learn step-by-step how to establish unidirectional and bidirectional HRMP channels between rollups and system rollups using XCM.
 template: index-page.html
 ---
 
 # Tutorials for Managing XCM Channels
 
 Establishing [XCM channels](/develop/interoperability/xcm-channels/) is essential to unlocking Polkadot's native interoperability. Before bridging assets or sending cross-chain contract calls, the necessary XCM channels must be established.
 
-These tutorials guide you through the process of setting up [Horizontal Relay-routed Message Passing (HRMP)](/develop/interoperability/xcm-channels/#establishing-hrmp-channels) channels for cross-chain messaging. Learn how to configure unidirectional channels [between parachains](/tutorials/interoperability/xcm-channels/para-to-para/) and the simplified single-message process for bidirectional channels with [system parachains like Asset Hub](/tutorials/interoperability/xcm-channels/para-to-system/).
+These tutorials guide you through the process of setting up [Horizontal Relay-routed Message Passing (HRMP)](/develop/interoperability/xcm-channels/#establishing-hrmp-channels) channels for cross-chain messaging. Learn how to configure unidirectional channels [between rollups](/tutorials/interoperability/xcm-channels/para-to-para/) and the simplified single-message process for bidirectional channels with [system rollups like Asset Hub](/tutorials/interoperability/xcm-channels/para-to-system/).
```

Comment:
Should para-to-para be renamed? And para-to-system?

Reply:
Renamed